File: Java/Processor.java
public class Processor {
    private String name;    // menampung nama processor
    private int price;      // menampung harga

    // constructor
    public Processor() {
    }

    public Processor(String name, int price) {
        this.name = name;
        this.price = price;
    }

    // setter and getter
    public void setName(String name) {
        this.name = name;
    } 

    public String getName() {
        return this.name;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return this.price;
    }

    // method
    public void printProcessor() {
        System.out.println("Processor Name      : " + this.getName());
        System.out.println("Processor Price     : Rp " + this.getPrice() + ",-");
    }
}

File: C++/Processor.cpp
#include <iostream>
#include <string>

using namespace std;

class Processor
{
    private:
        string name;    // menampung nama processor
        int price;      // menampung harga
    
    public:
        // constructor
        Processor() {}

        Processor(string name, int price)
        {
            this->name = name;
            this->price = price;
        }

        // setter and getter
        void setName(string name)
        {
            this->name = name;
        }

        string getName()
        {
            return this->name;
        }

        void setPrice(int price)
        {
            this->price = price;
        }

        int getPrice()
        {
            return this->price;
        }

        // method
        void printProcessor()
        {
            cout << "Processor Name     : " << this->getName() << endl;
            cout << "Processor Price    : Rp " << this->getPrice() << ",-" << endl;
        }

        // destructor
        ~Processor() {}
};